packages/components/data-table/src/data-table.story.js

```diff
@@ -22,6 +16,11 @@ import NumberInput from '../../inputs/number-input';
 import IconButton from '../../buttons/icon-button';
 import { InformationIcon } from '../../icons';
 
+// aux function for the maxHeight and maxWidth prop knobs
+// which can accept Number values but the text knob always returns String
+const castToNumberIfPossible = (value) =>
```

Comment:
Should we maybe throw some unit tests at this?

Reply:
That fn is just for the storybook example, but I should add for this one:
https://github.com/commercetools/ui-kit/pull/1455/files#diff-bd664be2fdc62a0525a6bec77cb25841R6

In fact, I think I'll move it to another file (as an util) and test it individually.

Reply:
That sounds good. Thanks.